btrack/libwrapper.py

```diff
@@ -83,13 +83,13 @@ def load_library(filename):
     try:
         lib = ctypes.cdll.LoadLibrary(full_lib_file)
         logger.info(f"Loaded btrack: {full_lib_file}")
-    except IOError:
-        raise IOError(f"Cannot load shared library {full_lib_file}")
+    except OSError as err:
+        raise OSError(f"Cannot load shared library {full_lib_file}") from err
```

Comment:
I've never seen this before, why have it?
```suggestion
        raise OSError(f"Cannot load shared library {full_lib_file}")
```

Reply:
if you catch an exception and then raise another, you generally want to chain them so that the user can see the link between exceptions, check out https://docs.python.org/3/tutorial/errors.html#exception-chaining

Reply:
yeah it should be really